docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,6253 @@
+{
```

Comment:
<p>Can you also add a plot overlaying both together with different colors and labels? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>